.circleci/config.yml

```diff
@@ -148,7 +148,16 @@ jobs:
                  echo_green "=> Package check OK"
              done
 
+             make \
+                 -f .config \
+                 -f tmp/.packagedeps \
+                 -f <(echo '$(info $(sort $(package-y) $(package-m)))'; echo -en 'a:\n\t@:') \
+              | tr ' ' '\n' >enabled-package-subdirs.txt
              for PKG in $PKGS ; do
+                 if ! grep -qE "(^|/)$PKG$" enabled-package-subdirs.txt; then
+                        echo_red "===+ Building: $PKG skipped.  It cannot be enabled with $SDK_FILE"
```

Comment:
I see 2 spaces before "It"

Reply:
Haha, that's just old habits (but I am not [over 40](https://www.cultofpedagogy.com/two-spaces-after-period/))

Reply:
Removed, period.